README.md

```diff
@@ -26,9 +26,9 @@ $ npm install basic-auth
 var auth = require('basic-auth')
 ```
 
-### auth(req)
+### auth(request)
 
-Get the basic auth credentials from the given request. The `Authorization`
+Get the basic auth credentials from the given [request](https://nodejs.org/dist/latest/docs/api/http.html#http_class_http_incomingmessage). The `authorization`
```

Comment:
I would keep the header name here as `Authorization` as it's the "canonical" casing of a HTTP header name, which is what this sentence is referring to, not the name of a JavaScript object key.